packages/core/src/Input/Input.tsx

```diff
@@ -402,6 +402,13 @@ function Input<T extends string | NumberInputType>({
 		[onChange, onValueChange, type]
 	)
 
+	// Prevent the mouse wheel from increasing/deacreasing the number
+	const handleWheel = (e: WheelEvent) => e.preventDefault()
```

Comment:
It seems that this type is not correct. I see an error when I open the file.

Reply:
I think I have a lint problem.  I re-structured it and it should be fine now.